ciphey/basemods/Decoders/tap_code.py

```diff
@@ -0,0 +1,57 @@
+# by https://github.com/RustyDucky
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+
+@registry.register
+class Tap_code(Decoder[str, str]):
+    def decode(self, input: T) -> Optional[U]:
+        try: #So it doesn't throw an error when its not the right format or just not tap code
+            medium = ' '.join(map(str, input)).split(" ") #Dealing with the format of user input
+            output = ""
+
+            xone = ["A", "B", "C/K", "D", "E"]
+            xtwo =  ["F", "G", "H", "I", "J"]
+            xthr =  ["L", "M", "N", "O", "P"]  #Making the tap code table
+            xfour = ["Q", "R", "S", "Y", "U"]

```

Comment:
"Y" instead of "T" - is this on purpose?

Reply:
We should move this into CipheyDIsts too